ports/esp32/boards/GENERIC_S2_SAOLA/mpconfigboard.cmake

```diff
@@ -0,0 +1,10 @@
+set(IDF_TARGET esp32s2)
+set(USB_ENABLED 0)
```

Comment:
Why does this need to be disabled here?  I would think it's enough to just "do nothing" and USB won't be enabled.  The board needs to include `boards/sdkconfig.usb` to enable USB.

Reply:
@dpgeorge Sorry for taking so long to follow up on this.
I'm not quite sure what my exact thought process was here -- but I have updated this PR to simply forgo including `boards/sdkconfig.usb` as you mentioned. 

Appears to work fine on 2 of my SAOLA-S1 devices as of the latest commit.